mapie/time_series_regression.py

```diff
@@ -156,6 +155,25 @@ def fit(
         method of ``MapieTimeSeriesRegressor`` computes the
         ``conformity_scores_`` with relative values.
 
+        Parameters
```

Comment:
this docstring should be above the __init__

Reply:
Of course! I am going to fix that.

Reply:
Or should I let it here and write a new doctring above the init?

Reply:
I'm not sure I understand your proposal because this docstring is specific to the `fit` method. Can you confirm that you want me to remove the docstring from the `fit` method and rewrite a new one for the `MapieTimeSeriesRegressor`class?

Reply:
my bad, the way the code was displayed was misleading. It's ok !